DesignSafe-CI/portal
# Bug/wc 84 metrics modal error

## Overview: ##
Prevent blank or null values from causing the Metrics Modal to crash

## PR Status: ##


## Related Jira tickets: ##

* [WC-84](https://tacc-main.atlassian.net/browse/WC-84)

## Summary of Changes: ##

## Testing Steps: ##
1. Find a recent publication that has 0 Downloads, 0 Views, 0 Citations. 
2. Click the Details button to open the modal
3. Make sure the modal loads. 

## UI Photos:
![Screen Shot 2024-09-05 at 1 48 29 PM](https://github.com/user-attachments/assets/17d03edf-6102-435b-a13a-9d9731cf8eb8)

## Notes: ##
